buildkite/bazelci.py

```diff
@@ -521,6 +521,13 @@
         "queue": "macos",
         "python": "python3",
     },
+    "macos_legacy": {
+        "name": "macOS",
+        "emoji-name": ":darwin: macOS",
+        "publish_binary": ["macos"],
```

Comment:
Leave publish_binary empty? I think we can keep the trusted machines in the macos queue (since it's a different org), and IIRC this field is only relevant for the trusted org?

Reply:
Indeed, done!